tests/test-errortracker.js

```diff
@@ -0,0 +1,251 @@
+/**
+ * Created by rigwanzojunior on 6/7/17.
+ */
+
+const express = require('express');
+const chai = require('chai');
+const chaihttp = require('chai-http');
+const mocha = require('mocha');
+const statusCodes = require('http-status-codes');
+const app = require('../app');
+const assert = require('assert');
+const sinon = require('sinon');
+const describe = mocha.describe;
+const before = mocha.before;
+const after = mocha.after;
+const beforeEach = mocha.beforeEach;
+const afterEach = mocha.afterEach;
+const expect = chai.expect;
+const it = mocha.it;
```

Comment:
Nit, please reorder the requires into the following:

1. Require calls (modules, `require('express')`)
2. Require calls (relative files, `require('../app')`)
3. general constants (like `before = mocha.before`)